client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -207,40 +341,59 @@ const EmailMailboxActionMenu = ( { account, domain, mailbox } ) => {
 		return null;
 	}
 
+	const removeMailboxConfirmationItems = getRemoveMailboxConfirmationActionItems(
+		dispatch,
+		domain,
+		mailbox,
+		onAccountChanged,
+		translate
+	);
+
 	return (
-		<EllipsisMenu position="bottom" className="email-mailbox-action-menu__main">
-			{ menuItems.map(
-				( {
-					href,
-					image,
-					imageAltText,
-					isInternalLink = false,
-					key,
-					materialIcon,
-					onClick,
-					title,
-				} ) => (
-					<PopoverMenuItem
-						key={ href || key }
-						className="email-mailbox-action-menu__menu-item"
-						isExternalLink={ ! isInternalLink }
-						href={ href }
-						onClick={ onClick }
-					>
-						{ image && <img src={ image } alt={ imageAltText } /> }
-						{ materialIcon && <MaterialIcon icon={ materialIcon } /> }
-						{ title }
-					</PopoverMenuItem>
-				)
-			) }
-		</EllipsisMenu>
+		<>
+			<RemoveMailboxConfirmationDialog
+				translate={ translate }
```

Comment:
There is no need to pass `translate`, we can use the `useTranslate()` hook in `RemoveMailboxConfirmationDialog` instead.